docs/source/development.md

```diff
@@ -29,13 +29,17 @@ docker build \
 Run this inside the `pipeline/` folder.
 
 ```
+pip3 install -r requirements.txt
 DB_USER=$DB_USER \
 DB_PASSWORD=$DB_PASSWORD \
 DB_HOST=$DB_HOST \
 DB_PORT=$DB_PORT \
 DB_DATABASE=$DB_DATABASE \
 GH_TOKEN=$GH_TOKEN \
-python3 -m unittest -v
+run coverage run --source scraper,model --omit */__init__.py,*/tests/* -m unittest
+coverage html
+open htmlcov/index.html
```

Comment:
Doesn't `open` only really work on macOS? Maybe delete this and the preceding line and just have people run `coverage report -m` like you say on line 42.